Reactions/ReactorNull.H

```diff
@@ -18,6 +18,8 @@ public:
 
   void close() override {}
 
+  void printFinalStats(void* mem) override {}
```

Comment:
since this is unused, can you `/*mem*/` ?

Reply:
done